mlops/parallelm/mlops/stats/stats_helper.py

```diff
@@ -55,6 +56,11 @@ def _set_classification_stat(self, name, data, model_id, timestamp, **kwargs):
                 AccuracyScore.get_mlops_accuracy_stat_object(accuracy_score=data)
             category = StatCategory.TIME_SERIES
 
+        elif name == ClassificationMetrics.AUC:
```

Comment:
The big if/elif/elif... will be replaces with registering a handler function of each stats. This will come in a separate PR after the initial 12 stats goes in.

Reply:
yes. i had put them in registry :)
pr: https://github.com/mlpiper/mlpiper/pull/93